### PR TITLE
Kdesktop 809 Add spacing between on UI and remove blank dotes in release notes

### PR DESCRIPTION
**Pull request by herve-er**

### CODE REVIEW

File: release_notes/kDrive-3.6.0.20240501/kDrive-3.6.0.20240501-de.html [hunk 1]
```diff
@@ -27,7 +27,6 @@ <h2><b>Neue Funktionen</b></h2>
     <li>Linux - Synchronisierung von Links (Symlinks)</li>
     <li>Schutz hinzugef&uuml;gt, um zu verhindern, dass 2 Synchronisierungen innerhalb desselben Ordners stattfinden</li>
     <li>Die Protokolle werden jetzt 72 Stunden statt 24 Stunden aufbewahrt.</li>
```
Comment (luc-guyot-infomaniak) [inline release_notes/kDrive-3.6.0.20240501/kDrive-3.6.0.20240501-de.html:29]: Regarding the client application logs, the purge frequency is 7 days and not 72 hours.
Regarding the server application logs, I don't know. @ChristopheLarchier @ClementKunz Is it parametrized in some configuration files?

Comment (herve-er) [reply release_notes/kDrive-3.6.0.20240501/kDrive-3.6.0.20240501-de.html]: I updated the log purge frequency from 72 hours to 7 days in release notes (see commit: https://github.com/Infomaniak/desktop-kDrive/pull/84/commits/da5f0fa6fcabd6f56c0d10f1b8d2078319136458).

Comment (ClementKunz) [reply release_notes/kDrive-3.6.0.20240501/kDrive-3.6.0.20240501-de.html]: IMHO both logs should depend on the same parameter

File: release_notes/kDrive-3.6.0.20240501/kDrive-3.6.0.20240501-win-es.html [hunk 2]
```diff
@@ -24,8 +24,7 @@ <h2><b>Novedades</b></h2>
   <ul>
     <li>Sincronizaci&oacute;n de enlaces (enlaces simb&oacute;licos y enlaces)</li>
     <li>Protecci&oacute;n a&ntilde;adida para evitar que se sincronicen 2 sincronizaciones dentro de la misma carpeta.</li>
-    <li>Ahora los registros se conservan durante 72 horas en lugar de 24.</li>
-    <li></li>
+    <li>Ahora los registros se conservan durante 7 d�as en lugar de 24.</li>
```
Comment (ClementKunz) [inline release_notes/kDrive-3.6.0.20240501/kDrive-3.6.0.20240501-win-es.html:27]: Special characters (like `í`) must be HTML encoded. We have a script for that (`encode.sh`)

Comment (herve-er) [reply release_notes/kDrive-3.6.0.20240501/kDrive-3.6.0.20240501-win-es.html]: Thanks, I updated it in commit: https://github.com/Infomaniak/desktop-kDrive/pull/84/commits/7b8621dacd74de81e04bace43ff7b04a4398d31a.

**Review by ClementKunz — APPROVED** [review]

**Pull request merged**